tests/extract_tests.ml

```diff
@@ -26,13 +26,11 @@ let disabled =
 
 let with_open_in fn f =
   let ic = open_in fn in
-  Fun.protect ~finally:(fun () -> close_in_noerr ic)
-    (fun () -> f ic)
+    f ic
```

Comment:
Since this is just for the test harness, we can do away with the `protect` call, but you should at least call `close_in ic` (and `close_out oc` below) in order to keep up appearances :)

Reply:
Also, the indentation seems a bit off.